pkg/pagerduty/pagerduty.go

```diff
@@ -126,6 +128,77 @@ func (p PagerDuty) AddNote(incidentID string, noteContent string) error {
 	return nil
 }
 
+// ExtractExternalIDFromCGHMAlertBody extracts from an Alert body until an external ID
+// the function is a member function but doesn't use any of the types
+func (_ PagerDuty) ExtractExternalIDFromCGHMAlertBody(data map[string]interface{}) (string, error) {
```

Comment:
Can we come up with a Smaller function name ?

Reply:
What about ExtractExternalID? Or ExtractExternalCGHMAID? If we look on the receiver of the function, I think it becomes clear that this function is about alert bodys, but I never worked with PagerDuty. You might come up with a better idea :)

Reply:
nope, that's a good idea, naming is hard :/